lms/static/sass/views/_shoppingcart.scss

```diff
@@ -69,6 +69,11 @@ $light-border: 1px solid $gray-l5;
     vertical-align: top;
   }
 
+  img {
```

Comment:
Can we switch this to use class or something? 
It's a bit too general to just set these properties for all img elements on all shopping cart pages (although it's just kind of a reset).